src/fpre.rs

```diff
@@ -196,6 +196,7 @@ pub(crate) async fn fpre(channel: &mut (impl Channel + Send), parties: usize) ->
     Ok(())
 }
 
+#[cfg(not(feature = "is_sync"))]
```

Comment:
This can also be done as `#[cfg(all(test, not(feature = "is_sync")))]` to have both cfgs in one expression. I'm not sure which I prefer.